deploy.sh

```diff
@@ -1,5 +1,30 @@
 #!/bin/bash
 
+# Node should be installed
+
+if (which node)
+then 
+ echo "node is installed"
+else
+ echo "installing node"
+ brew install node
+fi
+
+# https://github.com/nodesource/distributions#deb
+
+
 # Npm should be installed
+# gets installed with brew install node
+
+if (npm -version)
+then 
+ echo "npm is installed"
+else
+ echo "installing npm"
+ brew upgrade node
```

Comment:
You have to upgrade node to get npm? Is npm already installed with node?

Reply:
Yes it gets installed when you install node. So this code block is not even needed, I believe? I wrote to get your thoughts? When you upgrade node, it also upgrades npm.

Reply:
Yeah just remove this block.

Reply:
That's a bet, I'll do it when I get by my machine.